pkg/lclient/fakeclient.go

```diff
@@ -39,6 +39,74 @@ var mockImageSummary = []types.ImageSummary{
 	},
 }
 
+var mockContainerJSONList = []types.ContainerJSON{
+	types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "/node",
+			Image: "node",
+			ID:    "1",
+		},
+		Mounts: []types.MountPoint{
+			{
+				Destination: OdoSourceVolumeMount,
+			},
+		},
+		Config: &container.Config{
+			Image: "node",
+			Labels: map[string]string{
+				"component": "test",
+				"alias":     "alias1",
+			},
+		},
+	},
+	types.ContainerJSON{
+		ContainerJSONBase: &types.ContainerJSONBase{
+			Name:  "/go-test",
```

Comment:
You need to update the config labels for these too

Reply:
Why? It's not related to anything we've modified so far in this. :thinking: 

Also, the tests are complaining elsehwere:
```
Applying URL changes                                                                           
 ✓  URL example: https://example.com created
                                                                                               
Applying URL changes                                                                           
 ✓  URL example: https://example.com created                                                   
--- FAIL: TestListDockerURL (0.00s)                                                            
    url_test.go:1471: Expected error since no physical env file to write   
    url_test.go:1476: Expected error since no physical env file to write                       
    --- FAIL: TestListDockerURL/Case_1:_Successfully_retrieve_the_URL_list (0.00s)             
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1f22172]
                                               
goroutine 73 [running]:              
testing.tRunner.func1.1(0x2091700, 0x3455d90)                                                  
        /usr/lib/golang/src/testing/testing.go:940 +0x421                                      
testing.tRunner.func1(0xc000223680)                                                            
        /usr/lib/golang/src/testing/testing.go:943 +0x600               
panic(0x2091700, 0x3455d90)                                                                    
        /usr/lib/golang/src/runtime/panic.go:975 +0x3e3               
github.com/openshift/odo/pkg/url.ListDockerURL(0xc000560b20, 0x22673bc, 0x6, 0xc00007fbc0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/dshah/go/src/github.com/openshift/odo/pkg/url/url.go:528 +0xc92
github.com/openshift/odo/pkg/url.TestListDockerURL.func1(0xc000223680)                         
        /home/dshah/go/src/github.com/openshift/odo/pkg/url/url_test.go:1528 +0xd1
testing.tRunner(0xc000223680, 0xc000560f40)
        /usr/lib/golang/src/testing/testing.go:991 +0x1ec
created by testing.(*T).Run
        /usr/lib/golang/src/testing/testing.go:1042 +0x661
FAIL    github.com/openshift/odo/pkg/url        0.758s
ok      github.com/openshift/odo/pkg/url/labels (cached)
--- FAIL: TestGetAbsPath (0.00s)
    util_test.go:513: Running test:  Case 1: Valid abs path resolution of `~`
    --- FAIL: TestGetAbsPath/Case_1:_Valid_abs_path_resolution_of_`~` (0.00s)
        util_test.go:538: Expected /home/dshah/, got /home/dshah
    util_test.go:513: Running test:  Case 2: Valid abs path resolution of `.`
FAIL
FAIL    github.com/openshift/odo/pkg/util       0.504s
?       github.com/openshift/odo/pkg/version    [no test files]
ok      github.com/openshift/odo/pkg/watch      (cached)
FAIL
make: *** [Makefile:144: test] Error 1
```

Reply:
> Why? It's not related to anything we've modified so far in this. :thinking: 

Ah, my bad.